.pylintrc

```diff
@@ -22,8 +22,7 @@ ignore-patterns=
 #init-hook=
 
 # Use multiple processes to speed up Pylint.
-# jobs=1
-jobs=2
+jobs=1
```

Comment:
Is it no longer true you can use `jobs=2`? If so could you update the comment to explain why?

Reply:
Yeah, it caused a race condition which ignored the `min-similarity-lines` setting.